submission/dummy_method.py

```diff
@@ -1,7 +1,15 @@
 import gym
 import numpy as np
-from sequoia import Method, ClassIncrementalSetting, PassiveEnvironment, Observations, Actions, Environment, Rewards
 import tqdm
+from sequoia import (
+    Actions,
+    ClassIncrementalSetting,
+    Environment,
+    Method,
+    Observations,
+    PassiveEnvironment,
+    Rewards,
+)
```

Comment:
Is this Black formatting? Which line length?

Reply:
Yes, line length is at the default (100 I think?)